packages/@aws-cdk/aws-cloud9/README.md

```diff
@@ -49,3 +49,17 @@ const c9env = new cloud9.Ec2Environment(this, 'Cloud9Env3', {
 new cdk.CfnOutput(this, 'URL', { value: c9env.ideUrl });
 ```
 
+### Cloning Repositories
+
+Use `clonedRepositories` to clone AWS Codecommit repositories into the environment:
+
+```ts
+  const repo = codecommit.Repository.fromRepositoryName(stack, 'Repo', 'foo');
+
+  new cloud9.Ec2Environment(stack, 'C9Env', {
+    vpc,
+    clonedRepositories: [
+      cloud9.CloneRepository.fromCodeCommit(repo, '/foo'),
```

Comment:
Use more meaningful value instead of `/foo` here to make it clearer what this parameter is. For example `/src`.